data-loader/build.gradle

```diff
@@ -1,4 +1,8 @@
 subprojects {
+
```

Comment:
```suggestion
```

Reply:
I have removed the empty line.
Thank you